src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -274,6 +279,115 @@ static bool has_32bit_output(const std::shared_ptr<ngraph::Node>& node) {
 inline bool has_32bit_input(const std::shared_ptr<ngraph::Node>& node) {
     return is_activation(node) || is_pooling(node);
 }
+
+static bool is_op_supported(const std::shared_ptr<ov::Node>& node,
+                            const std::string& gnaCompileTarget,
+                            const GNAPluginNS::Config config,
+                            bool raise_exception = false) {
```

Comment:
```suggestion
                            const bool allow_to_raise_exception_instead_of_retrurn_false = false) {
```
`raise_exception` is a bit misleading now,

Reply:
Renamed to `is_exception_allowed`